src/main/webapp/script.js

```diff
@@ -301,35 +301,81 @@ function createNotificationsElement(notification) {
  * Each element corresponds to a question to be displayed in the DOM.
  */
 function createQuestionElement(question, hasRedirect) {
-  const questionElement = document.createElement('li');
-  questionElement.setAttribute('class', 'list-group-item');
+  const questionWrapper = document.createElement('div');
+  questionWrapper.setAttribute('class', 'list-group-item');
+
+  const questionElement = document.createElement('div');
+  questionElement.setAttribute('class', 'media');
+  questionElement.setAttribute('style', 'width: auto');
```

Comment:
I think we should abstain from setting style with HTML, but instead modify the class in the CSS file.

Reply:
the thing is, since it is a bootstrap class, I'm not sure if I can add or override styles from it using CSS